### ISSUE TITLE
There should be timeout to avoid the flow being in stuck in 'In Progress' state

### ISSUE THREAD

**Issue by pkazlenka** [telstra/open-kilda#4983]
This is a follow-up of #4972. Where due to some bug or environment issue flow/y-flow update is stuck in 'In Progress' state, there is no way to call another update/delete of this flow (without direct intervention into Flow DB table).

Steps to reproduce:
1. Clone, build and start openkilda v1.12.1
2. Follow steps from #4972
3. Try to delete a flow
Expected: Flow is deleted
=> Flow can't be deleted because it is 'In Progress' state (eternally)
Optional steps to reproduce: probably, you can just update flow entry in OrientDB, setting 'status' field of the flow in flow/y_flow table to 'in progress'.

As a possible solution there could be putting timeout flow update method, after which changes are rolled back and 'update' call returns error or adding a scheduler which monitors flow statuses and enforces hanged flow to e.g. 'down' state.

**Comment by pablomuri** [comment]
But the timeout already exists and works, the **rollback** wasn't working because the affinityFlow was null. I wouldn't say I like the idea of having a scheduler because it can hide possible problems.

**Comment by niksv** [comment]
@pkazlenka Yes we need to starts with "What we want to see at the end". Generally any Flow must NOT stuck in "inprogress" state. We have we other cases when it stuck - we must to fix it. But I agree with @pablomuri that having some scheduler is not a good idea. We need to discuss it
